index.html

```diff
@@ -239,7 +239,15 @@ <h2>
       <p>
         For these reasons, the <a>user agent</a> SHOULD inform the user when
         the API is being used and provide a mechanism to disable the API
-        (effectively no-op), on a per-origin basis or globally.
+        (effectively no-op), on a per-origin basis or globally. Implementers
+        are encouraged to complement the normatively defined sticky
+        activation-based user activation-gating mitigation with the
```

Comment:
Encouraging implementers to implement normatively-defined things is weird. If they are normative then implementations should be doing them. This section probably needs a larger rewrite given that implementations currently do not inform the user when the API is in use either, or provide a mechanism to disable it.

Reply:
Thanks, yes indeed. I think it is clearer to simply remove the sentence starting with "Implementers are encouraged to ...".

And, considering the intent of this specification update is to specify what is currently implemented, the appropriate RFC 2119 term to use in this context is MAY. If implementations agree to add these additional mitigations, we will adjust the term accordingly.

These fixes are at https://github.com/w3c/vibration/pull/47/commits/130478720f49b3c73fcf5c86c06625fa64c7a743